spring-boot-modules/spring-boot-autoconfiguration/src/main/java/com/baeldung/autoconfiguration/MySQLAutoconfiguration.java

```diff
@@ -31,7 +31,13 @@
 import org.springframework.orm.jpa.vendor.HibernateJpaVendorAdapter;
 import org.springframework.util.ClassUtils;
 
-@Configuration
+import jakarta.persistence.EntityManagerFactory;
+
+/*@Configuration
```

Comment:
let's not forget to remove comment code from here

Reply:
@Bipinkumar27, indeed, are these comments needed here?